samples/deprecated/jobs.md

```diff
@@ -0,0 +1,84 @@
+# Jobs
```

Comment:
Trivial: May not be worthwhile but maybe we could add a comment at the start of the deprecated sample .md files that state this sample is deprecated with a link to the current service client outside of the deprecated folder. Might not be worth the trouble though since to see this, someone would have to come looking specifically in the deprecated folder....

On a side note, should we simply remove these from existence since they're just samples that we don't want anyone using?